_studio/mfx_lib/shared/src/mfx_h264_enc_common_hw.cpp

```diff
@@ -1399,6 +1399,7 @@ bool MfxHwH264Encode::IsMctfSupported(
     mfxExtCodingOption2 const & extOpt2 = GetExtBufferRef(video);
     isSupported = (IsOn(extOpt2.ExtBRC) &&
         IsExtBrcSceneChangeSupported(video) &&
+        (video.mfx.FrameInfo.Width <= 3840 && video.vpp.In.Height <= 2160) &&
```

Comment:
where this comes from? Do we have some MCTF limitations which we did not handle before?

Reply:
Yes, this is an actual limitation for MCTF, it can only denoise frames up to 3840x2160.

Reply:
Ok, that's fine then, but this needs to be yet another commit!